config/routes.rb

```diff
@@ -2,4 +2,9 @@
   devise_for :trainers
   root to: "home#index"
   resources :trainers
+  patch "home/index", to: "pokemons#capture", as: "capture"
+  patch "trainers/show.html.erb", to: "pokemons#damage", as: "damage"
+  get "pokemons/new", to: "pokemons#new", as: "new"
+  resources :pokemons, only: [:new, :create, :index]
+  post "pokemons/new", to: "pokemons#create"
```

Comment:
Typically creating a resource is a POST on `pokemons`, not `pokemons/new`, but this willstill work!